templateplugin/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,11 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
```

Comment:
This should mirror exactly what the client has. I would be fine with just not including this file to keep the template as clean as possible.

Reply:
I think the idea was to prevent the tests from writing to the .runelite logs dir right? If so this file would be required.

Reply:
Yes, but in practice there are < 5 plugins with tests so I'm fine with not worrying about this.

Reply:
New commit added that removes it. I'll add something to the wiki about this if it's merged.